subsys/net/ip/net_core.c

```diff
@@ -90,9 +99,12 @@ static inline enum net_verdict process_data(struct net_pkt *pkt)
 				net_stats_update_processing_error(
 							net_pkt_iface(pkt));
 			}
-
```

Comment:
same here, and actually it is be preferred that there is an empty line after closing block (so the original code is ok)